server/src/controller/user.ts

```diff
@@ -44,46 +98,42 @@ export class UserController {
         } else {
             // return a BAD REQUEST status code and error message
             ctx.status = 400;
-            ctx.body = "The user you are trying to retrieve doesn't exist in the db";
+            ctx.body = { errors: {error: "The user you are trying to retrieve doesn't exist in the db"}}
         }
-
     }
 
     /**
-     * POST /users
-     * 
-     * Create a user.
+     * GET /api/user/licks
+     *
+     * Get users a lick is shared with by id.
      */
-    public static async createUser(ctx: Context): Promise<void> {
+    public static async getAuthUserLicks(ctx: Context): Promise<void> {
 
-        // get a user repository to perform operations with user
         const userRepository: Repository<User> = getManager().getRepository(User);
+        // will always return the currently authenticated user
+        const authUser: User = await userRepository.findOne({ where: {id: (ctx.state.user.id)}, relations: ['licks']});
+        
+        ctx.status = 200; // OK
+        ctx.body = authUser.licks;
```

Comment:
could authUser be null?

Reply:
technically it could be, but ctx.state.user.id is always defined since this is a protected router, so something would have to be something go wrong on the .findOne() operation. I put in some checks though just incase and sent back a 500 if the user couldn't be found